pymc/model/fgraph.py

```diff
@@ -369,7 +377,7 @@ def clone_model(model: Model) -> Model:
 
     Recreates a PyMC model with clones of the original variables.
     Shared variables will point to the same container but be otherwise different objects.
-    Constants are not cloned.
+    Constants are not cloned and if guassian process variables are detected then a warning will be triggered.
```

Comment:
Let's not mention it, this is just a temporary thing we want to fix

```suggestion
    Constants are not cloned.
```

Reply:
Okay, I will make that change as well!